docs/hammerdb.md

```diff
@@ -5,9 +5,9 @@
 ## Running hammerdb
 
 Given that you followed instructions to deploy operator,
-you can modify [cr.yaml](../resources/crds/ripsaw_v1alpha1_hammerdb_cr.yaml) to your needs.
+you can modify [cr.yaml](../resources/crds/hammerdb_crds/ripsaw_v1alpha1_hammerdb_cr.yaml) to your needs.
 
-The pgbench workload needs to be pointed at an existing MS-SQL databases via the CR file.
+The pgbench workload needs to be pointed at an existing [MSSQL](../resources/crds/hammerdb_crds/mssql), [MARIADB](../resources/crds/hammerdb_crds/mariadb), [POSTGRES](../resources/crds/hammerdb_crds/postgres) databases via the CR file.
```

Comment:
it should read 'database via the CR file', singular vs. plural

Reply:
Thx for your comment.
Please let me know if thats OK now:
https://github.com/cloud-bulldozer/benchmark-operator/pull/569/commits/9fa068311f6f86a8b9c0a46240821b6153079259

Reply:
Fix it https://github.com/cloud-bulldozer/benchmark-operator/pull/569/commits/e3e07500b23e61a755978c1fad7bdbe6449976e1
ignore the wether, revert it back https://github.com/cloud-bulldozer/benchmark-operator/pull/569/commits/b35eca811a06dd7efca8dccdc660a3545b44416c